app/controllers/v0/api_controller.rb

```diff
@@ -3,6 +3,10 @@ module V0
   class ApiController < ApplicationController
     skip_before_action :authenticate_user!
 
+    def cors_preflight
+      head(:ok)
+    end
```

Comment:
We might need to revisit this when we add authentication component between vets-api and this data service, but this is good for now.